templates/ciena_soas_vlan_show.textfsm

```diff
@@ -0,0 +1,31 @@
+Value VLAN_ID (\d+)
```

Comment:
Is this template named wrong now? It shows that the correct naming template was deleted. Rename back to original name please.